dotnet-desktop-guide/net/winforms/controls-design/snippets/designer-properties-overview/vb/CompassRose.vb

```diff
@@ -0,0 +1,29 @@
+﻿Imports System.ComponentModel
+
+Public Class CompassRose
+
+    '<defaultvalue>
+    <DefaultValue(GetType(Directions), "North")>
+    Public Property PointerDirection As Directions = Directions.North
+
+    <DefaultValue(10)>
+    Public Property DistanceInFeet As Integer = 10
+    '</defaultvalue>
+
+    '<browsable>
+    <Browsable(False)>
+    Public Property IsSelected As Boolean
+    '</browsable>
```

Comment:
Are those closing tags important for the docs?

Reply:
Yes, they're the snippet ID so it pulls just the code between the tag